scene/resources/material.h

```diff
@@ -546,12 +600,23 @@ class BaseMaterial3D : public Material {
 
 	AlphaAntiAliasing alpha_antialiasing_mode = ALPHA_ANTIALIASING_OFF;
 
+	StencilMode stencil_mode = STENCIL_MODE_DISABLED;
+	int stencil_flags = STENCIL_FLAG_READ;
+	StencilCompare stencil_compare = STENCIL_COMPARE_EQUAL;
+	int stencil_reference = 1;
+
+	Color stencil_effect_color = Color(0, 0, 0, 1);
```

Comment:
```suggestion
	Color stencil_effect_color;
```
Default value not needed here